selfservice/flow/request.go

```diff
@@ -106,19 +105,7 @@ func MethodEnabledAndAllowed(ctx context.Context, flowName FlowName, expected, a
 		return errors.WithStack(ErrStrategyNotResponsible)
 	}
 
-	var ok bool
```

Comment:
Are you certain that removing this won't cause regressions?

Reply:
I have added a fallback to enable the code strategy, if passwordless is enabled, but the code strategy is not: https://github.com/ory/kratos/blob/8821e6b0c51f7fbec05e05712ec4d7b53647aba4/driver/config/config.go#L764-L772

But, since code is enabled by default, there shouldn't be many cases where this case (`code.enabled: false && code.passwordless_enabled: true`) is relevant, in the first place.